src/mock/MockData.cpp

```diff
@@ -74,7 +74,7 @@ std::vector<Player> MockData::players_ = {
     {"Ray Allen", 44, false, 18, 1996, 2014, 1300, 18.9, 4, "America", 2},
     {"Klay Thompson", 30, true, 8, 2011, 2019, 615, 19.5, 1, "America", 3},
     {"Kristaps Porzingis", 24, true, 4, 2015, 2020, 237, 18.1, 2, "Latvia"},
-    {"Shaquille O'Neal", 48, false, 19, 1992, 2011, 1207, 23.7, 6, "America", 4},
+    {"Shaquile O'Neal", 48, false, 19, 1992, 2011, 1207, 23.7, 6, "America", 4},
```

Comment:
For reasons I can't figure out, the vast majority of the instances of Shaq in this repository misspell his name.

In case the code intentionally uses two spellings, I've swapped them so that the primary spelling is correct.

Because of this content, unlike all the other commits, I haven't integrated the whitespace fixes in, but have left them as a distinct commit ... I'd expect to fold them.

Reply:
The player's name is just Shaquille O'Neal